### PR TITLE
Warnings fix 

### PR DESCRIPTION
**Pull request by OlegSubik**
- Warnings fix + small ui adjustments

### CODE REVIEW

File: src/components/Admin/UserPage.tsx [hunk 1]
```diff
@@ -99,7 +99,7 @@ export const UserPage: FC<UserPageProps> = ({ mode = EditMode.readOnly, user, ti
 
   const handleSave = (user: UserModel) => {
     // Convert UserModel to UserInput
-    const { id: userID, profile, memberof, ...rest } = user;
+    const { id: userID, profile, ...rest } = user;
```
Comment (atstojanov) [inline src/components/Admin/UserPage.tsx:102]: `memberof` should not be present in the user input, will fail the GraphQL validation.
Saving user will throw an error:
`Field "memberof" is not defined by type "UserInput".`

File: src/components/Admin/UserPage.tsx [hunk 2]
```diff
@@ -130,6 +130,7 @@ export const UserPage: FC<UserPageProps> = ({ mode = EditMode.readOnly, user, ti
         },
       });
     } else if (isEditMode && user.id) {
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
       const { email, ...userToUpdate } = userInput;
```
Comment (atstojanov) [inline src/components/Admin/UserPage.tsx:134]: The `email` extraction can be omitted. 
The back-end now checks if the `email` is different than the existing one, not only if it is in the input data. Please remove that and pass the `userInput` directly to the mutation.

**Review by atstojanov — DISMISSED** [review]
There are comments that needs to be resolved.

File: src/components/Admin/UserPage.tsx (hunk 2 above)
Comment (OlegSubik) [reply src/components/Admin/UserPage.tsx:134]: done

File: src/components/Admin/UserPage.tsx (hunk 1 above)
Comment (OlegSubik) [reply src/components/Admin/UserPage.tsx:102]: done

**Review by techsmyth — APPROVED** [review]

**Pull request merged**